ukvm/ukvm-core.c

```diff
@@ -479,7 +479,7 @@ static void setup_cpuid(int kvm, int vcpufd)
         err(1, "KVM_SET_CPUID2 failed");
 }
 
-
+#if 0
```

Comment:
I think we will never ever use this functionality. You might as well delete it. If we want it back we have the commits history.